liqd_product/assets/scss/_layout.scss

```diff
@@ -1,3 +1,85 @@
+// should be used with ul/ol
+@mixin grid-tiles($n, $settings: ()) {
+    @include clearfix;
+
+    > * {
+        @include grid-same-width($n, $settings);
+    }
+
+    @supports (display: grid) {
+        display: grid;
+        grid-column-gap: _grid-get('gutter', $settings);
+        grid-template-columns: repeat($n, 1fr);
+
+        > * {
+            margin-left: 0 !important;
+            margin-right: 0 !important;
+            width: auto !important;
+        }
+    }
+}
+
+.l-wrapper {
+    @include clearfix;
+    position: relative;
+    width: 100%;  // required when used in a flex container
+    max-width: $wrapper-width;
```

Comment:
why didn't you just reduce the wrapper-width via _variables to make the page more smaller?

Reply:
Yeah. ¯\(º_o)/¯